src/styles/calendar.scss

```diff
@@ -0,0 +1,50 @@
+.calendar {
+  padding: 10px;
+
+  &__container {
+    display: flex;
+    gap: $gap;
+    flex-wrap: wrap;
+    list-style: none;
+    padding-left: 0;
+    width: (7 * $square-size) + 6 * $gap;
+
+    @for $day from 1 through 31 {
+      & :nth-child(n + #{$day}):before {
+        content: "#{$day}";
+        font-size: $day-font-size;
+        font-family: $day-font;
```

Comment:
```suggestion
      & :nth-child(n + #{$day}):before {
        content: "#{$day}";
```
You can just use calendar__day for other properties
![image](https://github.com/mate-academy/layout_calendar/assets/96409563/8b104f82-bd96-4a51-ba8d-638ac31deee6)

Reply:
fixed